package.json

```diff
@@ -15,26 +15,22 @@
   "bugs": {
     "url": "https://github.com/atlassian/react-beautiful-dnd/issues"
   },
-  "main": "lib/index.js",
-  "module": "esm/index.js",
+  "main": "dist/react-beautiful-dnd.cjs.js",
```

Comment:
is it common to do this rather than use a seperate folder?

Reply:
Dumb question, but cjs = CommonJS? 😊

Reply:
I think it is not necessary to create a folder for only one file in it. We have five file in the end which look very nice in one folder.

Yes, esm and cjs are the most popular shorthands.